frameworks/C++/libsniper/libsniper_bench/src/main.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020, RTBtech, MediaSniper, Oleg Romanenko (oleg@romanenko.ro)
+ */
+
+#include <sniper/event/Loop.h>
+#include <sniper/event/Timer.h>
+#include <sniper/http/Server.h>
+#include <sniper/log/log.h>
+#include <sniper/std/check.h>
+#include <sniper/std/vector.h>
+#include <sniper/threads/Stop.h>
+#include <thread>
+
+using namespace sniper;
+
+class Server
+{
+public:
+    explicit Server()
+    {
+        for (unsigned i = 0; i < std::thread::hardware_concurrency(); i++)
+            _workers.emplace_back(&Server::worker_noexcept, this, i);
+    }
+
+    ~Server()
+    {
+        for (auto& w : _workers)
+            if (w.joinable())
+                w.join();
+    }
+
+private:
+    void worker_noexcept(unsigned int thread_num) noexcept
+    {
+        try {
+            worker(thread_num);
+        }
+        catch (std::exception& e) {
+            log_err(e.what());
+        }
+        catch (...) {
+            log_err("[Server] non std::exception occured");
+        }
+    }
+
+    static string gen_date()
+    {
+        return fmt::format("Date: {:%a, %d %b %Y %H:%M:%S} GMT\r\n", fmt::gmtime(time(nullptr)));
+    }
+
+    void worker(unsigned thread_num)
+    {
+        auto loop = event::make_loop();
+        check(loop, "[Server] cannot init event loop");
+
+        http::server::Config config;
+        config.max_conns = 100000;
+        config.connection.keep_alive_timeout = 10min;
+        config.connection.request_read_timeout = 10s;
+
+        http::Server http_server(loop, config);
+        check(http_server.bind("", 8090), "[Server] cannot bind to localhost:8090");
+
+        string date = gen_date();
+        http_server.set_cb_request([&, this](const auto& conn, const auto& req, const auto& resp) {
```

Comment:
Thanks for your contribution @OlegRomanenko 

I don't see any type of routing here. Please see rule 6 in the [General Requirements](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#general-test-requirements)

> Some form of request routing is required, even if only a single test type (e.g., Plaintext) is implemented. In most cases, the framework's canonical router or a mainstream router library will be used. In some cases, it is considered normal and sufficiently production-grade to use hand-crafted minimalist routing using control structures such as if/else branching. This is acceptable where it is considered canonical for the framework.

Reply:
Oh sorry! I forgot about routing. We do not use routing in production.
Fixed.